recipes/trollsift/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "trollsift" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "4f67522224714e4c1ca612c82c5a1600c55fef879efbe1f30bcbe8f79e442ed0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Can this be noarch?

Reply:
Done. Thanks for the suggestion. I should have thought of that.